subiquity/server/controllers/filesystem.py

```diff
@@ -1405,6 +1409,20 @@ async def run_autoinstall_guided(self, layout):
                 disk_id=gap.device.id, gap=gap, allowed=[]
             )
 
+        reset_partition = False
+        reset_partition_size = None
+        rp_input = layout.get("reset-partition", None)
+        if rp_input:
+            reset_partition = True
+            if type(rp_input) in (str, int):
+                reset_partition_size = int(human2bytes(rp_input))
+                log.info(
+                    "autoinstall: will install reset partition"
+                    f"of size {reset_partition_size}"
```

Comment:
looks like we're missing a space :)

Reply:
Added a space in the end of the previous line, thanks!